openmetadata-ui/src/main/resources/ui/src/generated/entity/data/glossaryTerm.ts

```diff
@@ -246,6 +254,14 @@ export enum LabelType {
   Propagated = 'Propagated',
 }
 
+/**
+ * Label is from Tags or Glossary.
+ */
+export enum Source {
```

Comment:
We should rename this enum to `TagSource` to avoid confusion. Doesn't have to be in this patch. We can also take care of it in the next patches.

Reply:
Auto generated from json schema as it is there